miqa/learning/nn_training.py

```diff
@@ -588,7 +588,7 @@ def train_and_save_model(df, count_train, save_path, num_epochs, val_interval, o
         logger.info('-' * 25)
         logger.info(f'epoch {epoch + 1}/{num_epochs}')
         model.train()
-        epoch_loss = 0
+        epoch_loss = float(0)
```

Comment:
Another way to write this is as a literal, `0.0`, which would avoid the cast

Reply:
Changed in [107d673](https://github.com/OpenImaging/miqa/pull/634/commits/107d673714769932d133942ea8355def62151cc7)